### DISCUSSION TITLE
Convert account to PRO account for sdk.vercel.ai

### DISCUSSION THREAD

**Discussion by timtischler** [vercel/community#2978] [Help] [answered]
### Summary

I started out by creating a hobby account attached to my github account.  It's moving beyond being a hobby AND I want access to the pro features on sdk.vercel.ai.   I have created and upgraded teams to PRO, but my account remains hobby and it looks like the sdk site only looks at the permissions of the user. I deleted my account and recreated it with the idea of making it but it still insists my github linked account is HOBBY.  

Help!!!

### Steps to Reproduce

Look at the user for this email address, and then look at the owned team and the team member that's been invited.

**Comment by amyegan** [comment, marked as answer]
Hi @timtischler. When you upgrade to Pro, your personal Hobby account will still exist as a member of the new Team. There is a dropdown selector at the top of your Dashboard that lets you switch between your personal and team pages.

<img src="https://github.com/vercel/community/assets/8247105/cd9bb4bd-e02a-466e-a628-5780bad368ad" alt="team selector at top of Vercel dashboard" />

If your project currently belongs to your Hobby account, you will need to transfer it to the Team to make use of Pro features and limits. You can learn how to transfer your project in our documentation about [transferring a project](https://vercel.com/docs/concepts/projects/overview#transferring-a-project) and in our guide titled ["How do I transfer my Vercel project from a personal account to a team?"](https://vercel.com/guides/transferring-projects-from-hobby-to-team)

**Comment by alexpwrd** [comment]
I have the same issue, even though I created a team and upgraded to pro, I am unable to access pro models in the https://sdk.vercel.ai/

**Reply by amyegan** [reply]
Did you follow the [getting started with Vercel AI SDK](https://sdk.vercel.ai/docs/getting-started) steps to create your own project? Please share your project ID or latest deployment URL so I can look into it 🙏
